CREATE.md

```diff
@@ -142,6 +142,19 @@ use '~/projects/cool_name'
 require("lazy").setup({
     { dir = '/home/your_username_here/projects/cool_name', lazy = true },
 }
+
```

Comment:
Can you please split this into a separate code block for each file, so its more obvious they're different files, and include the example filename/path.

Actually lets just pull all the wordy bits out as regular text too (I know this is a bit different to what I said before in your previous issue :pray: ), eg

Alternatively, if you organize your plugins by having a seperate file for each...
```lua
-- ~/.config/nvim/init.lua
<init.lua content>
```

And treat Lush and your colorscheme...

```lua
-- ~/.config/nvim/lua/plugins/lush.lua (or whatever it should be)
<plugin contents>
```